dockerhub/debian-slim/Dockerfile

```diff
@@ -77,6 +77,8 @@ RUN groupadd bun \
       --shell /bin/sh \
       --create-home \
     && ln -s /usr/local/bin/bun /usr/local/bin/bunx \
+    && apt-get install -qq --no-install-recommends \
```

Comment:
did you manually test this works? i remember needing to do `apt-get update` in certain cases

Reply:
did now